be/src/olap/rowset/segment_v2/binary_dict_page.cpp

```diff
@@ -37,7 +37,8 @@ BinaryDictPageBuilder::BinaryDictPageBuilder(const PageBuilderOptions& options)
     // TODO: the data page builder type can be created by Factory according to user config
     _data_page_builder.reset(new BitshufflePageBuilder<OLAP_FIELD_TYPE_INT>(options));
     PageBuilderOptions dict_builder_options;
-    dict_builder_options.data_page_size = _options.dict_page_size;
+    dict_builder_options.data_page_size = _options.data_page_size;
```

Comment:
std::min(datapagesize:dict page size)? If user set data page size larger than dict page size??

Reply:
sure, will do just in case, though I do not know in which case will a user do that.

Reply:
> std::min(datapagesize:dict page size)? If user set data page size larger than dict page size??

in this case, we still need to use dict_page_size, see `is_page_full()`, for a dict page, we use dict_page_size to judge if it's full.

Reply:
> > std::min(datapagesize:dict page size)? If user set data page size larger than dict page size??
> 
> in this case, we still need to use dict_page_size, see `is_page_full()`, for a dict page, we use dict_page_size to judge if it's full.

instead, we should always use the smaller one when init. dict_page_size is only used for judging is_page_full.